### PR TITLE
Add QcLang

### PR DESCRIPTION
**Pull request by MymaQc**
## Adding a language

- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme
- [x] I have no association with the language
- [x] There are no copyright issues with this code
- [x] The language has not been added prior to this pull request

#### Link to programming language: 
https://github.com/RestartFU/qc-lang

### CODE REVIEW

File: q/Qc.qc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package main
+
+foreign import "stdio.h"
+
+func main() {
+    foreign use "C" {
+        printf("Hello, World!");
```
Comment (MrBrain295) [inline q/Qc.qc:7]:
```suggestion
        printf("Hello World");
```

**Review by MrBrain295 — COMMENTED** [review]
Is there a way to run this natively without C?

File: q/Qc.qc (hunk 1 above)
Comment (MymaQc) [reply q/Qc.qc:7]: Not at the moment, no

**Review by gnuhead-chieb — COMMENTED** [review]
Github repository of this langauge removed so I think this PR should be closed.

**Pull request closed without merging**